### DISCUSSION TITLE
Allow re-theming (or just make readable) red/black dialogs.

### DISCUSSION THREAD

**Discussion by DBPreston** [OpenRCT2/OpenRCT2#22250] [Feature requests]
Dialogs like, for example, acknowledging a screenshot has been saved, are black on red and very much lacking in contrast. Similarly error messages like when a ride or stall has no path leading to it which are shown at the bottom of the screen, and are red on black, also are difficult to read if one's eyesight is less than perfect. 

These should be improved.

**Comment by ZeeMaji** [comment]
For what it's worth, i believe RCT1 used white on red which might be more readible.

**Comment by Gymnasiast** [comment]
For the error dialogue, this is not difficult to pull off:
![](
https://private-user-images.githubusercontent.com/1478678/351515244-a7582414-0944-4ef3-99c7-9cf2d790c6a2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE3NzUyOTcsIm5iZiI6MTcyMTc3NDk5NywicGF0aCI6Ii8xNDc4Njc4LzM1MTUxNTI0NC1hNzU4MjQxNC0wOTQ0LTRlZjMtOTljNy05Y2YyZDc5MGM2YTIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDcyMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA3MjNUMjI0OTU3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZWJjMzdkNTMwYzMxYjViZDg0MDAxZjc4MDE0YWQzMGY5ODEyZDczNzE0ZTdiNGE2OTcyY2VhODk2ZTMyZWFkMiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.2TTs7DjIK5zGDwjgpSpM6_EBdimKwDWvcCx1dcTVM5A)

I created https://github.com/OpenRCT2/OpenRCT2/pull/22357 for that.

The other is a bit more difficult.

**Comment by AaronVanGeffen** [comment]
Addressed in #22357.

**Reply by Gymnasiast** [reply]
Well, half of it - the other issue is news messages, which still need addressing.
